tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = cov-erase, flake8, {pypy27,pypy36,pypy37,py27,py35,py36,py37,py38,py39}-{tests,flake8,check-manifest,check-newsfragment}, cov-report
+envlist = cov-erase, flake8, {pypy27,pypy37,pypy38,py27,py35,py36,py37,py38,py39,py310}-{tests,flake8,check-manifest,check-newsfragment}, cov-report
```

Comment:
I must say that I don't quite know why he separation between `cov-erase` and `cov-report`.

In my projects I just join the testing and coverage to one *Tox* task. Here's an [example](https://github.com/saroad2/statue/blob/main/tox.ini)